aws/services/CloudFormation/MacrosExamples/StackMetrics/lambda/cfnresponse.py

```diff
@@ -11,14 +11,16 @@
 SUCCESS = "SUCCESS"
 FAILED = "FAILED"
 
-def send(event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False):
+def send(event, context, responseStatus, responseData, reason=None, physicalResourceId=None, noEcho=False):
```

Comment:
Suggest to keep existing signature by adding reason as last optional argument. 
That allows to keep existing implementations safe and w/o side-effects.

```suggestion
def send(event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False, reason=None):
```

Reply:
Great point!